tools/ld-chroma-decoder/main.cpp

```diff
@@ -229,7 +239,7 @@ int main(int argc, char *argv[])
     parser.addPositionalArgument("input", QCoreApplication::translate("main", "Specify input TBC file (- for piped input)"));
 
     // Positional argument to specify output video file
-    parser.addPositionalArgument("output", QCoreApplication::translate("main", "Specify output RGB file (omit or - for piped output)"));
+    parser.addPositionalArgument("output", QCoreApplication::translate("main", "Specify output file (omit or - for piped output). RGB48, YUV444P16, GRAY16 pixel formats are supported."));
```

Comment:
If you had an output-format option, the list of formats here could go into the help for that.